Files/UserControls/MultitaskingControl/BaseMultitaskingControl.cs

```diff
@@ -41,7 +41,7 @@ public BaseMultitaskingControl()
 
         public ObservableCollection<TabItem> Items => MainPageViewModel.AppInstances;
 
-        public List<ITabItem> RecentlyClosedTabs { get; private set; } = new List<ITabItem>();
+        public static List<ITabItem> RecentlyClosedTabs { get; private set; } = new List<ITabItem>();
```

Comment:
Ignore this change, it fixes an issue with "Ctrl+Shift+T" but it's unrelated to this PR.